Add example on colors of calculation properties (#141)
port-labs/port-docs: docs/software-catalog/blueprint/calculation-properties.md

}
```

## Colorized Calculation Properties

> It is possible to colorize Calculation Properties based on their value. This is done by adding a `colorized` key with the value `true` to the Calculation Property object. You can also add a `colors` key to specify the colors for the different values, otherwise the colors will be chosen automatically.

Reviewer: ```suggestion
You can colorize calculation properties according to their value, by adding a `colorized` key with the value `true` to the calculation property object. You can also add a `colors` key to specify the colors of the different values, otherwise, the colors will be chosen automatically for you.
```
